docs/docs/installation/configuring-superset.mdx

```diff
@@ -99,7 +99,7 @@ If you're not using Gunicorn, you may want to disable the use of `flask-compress
 If you are running superset behind a load balancer or reverse proxy (e.g. NGINX or ELB on AWS), you
 may need to utilize a healthcheck endpoint so that your load balancer knows if your superset
 instance is running. This is provided at `/health` which will return a 200 response containing “OK”
-if the the webserver is running.
+if the webserver is running.
```

Comment:
This is a doubled word, it's a fairly new feature of my tooling, and I'm generally fairly happy w/ the items it flags.

Reply:
nice!